### DISCUSSION TITLE
Effective and Total Stress

### DISCUSSION THREAD

**Discussion by MiguelSC98** [Anura3D/Anura3D_OpenSource#141] [Q&A] [answered]
Dear all.
I'm modelling a horizontal soil layer under two conditions to evaluate effective and total stress.
1st Condition
One layer of dry soil (Height: 20 m)
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/12fe410c-4593-41a1-beb1-6cbd28767cdb)
Calculation data:
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/77a73a61-d4ec-45ac-a6d4-dddd4f4cb798)
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/d153d3f2-e961-4e18-843b-d7751b34224f)
2nt Condition
Dry soil (deph: 0-15 m) and saturated soil (depht: 15-20 m) under phreatic surface (deph;15 m).
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/a181dd68-a5f1-4c9a-9ab8-b15fc0ccb66e)
Phreatic surface:
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/92004279-e65f-478d-91ab-3054bf51354e)
Calculation data:
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/1f242e1f-cd72-4fb2-9f90-973eddd2f0d3)
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/468a2b95-735a-440e-ba32-dc1af61de4bb)

Convergencia Criteria in both conditions: Quasi-static convergence
Parameters:
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/b3318ca7-003b-4180-b99e-1362fee40ecf)

Results

**1st Condition (Dry condition)**

Efective and Total Stresess are the same (maximum  value 374 kPa)
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/c9336e25-8ff9-4f42-b654-25ecf9ed1eb4)
Weight_solid plot just shows an unique value (19.62 kN/m3) 
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/230769b7-39a1-4789-8a87-cf7e0b287b17)

**2nt Condition (Dry and saturared soils)**

Efective and Total Stresess are the same (maximum  value 354 kPa)
Total stress: ![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/154d7bfa-05dc-4b7f-8045-b6f298d37389)
Effective stress: ![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/f935c000-f633-4d0f-9a39-1dfcc6c95316)
Stress liquid and pressure _liquid plots show null values in the model
Weight_liquid plot is acoording with input data of model
Weight_solid plot just shows an unique value (19.62 kN/m3)  despite two materials (dry and satured soils) have been modelled
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/121762035/5bdc53a0-cd1b-4d55-b700-3d8bf59f613b)

Questions:
Is weight_solid plot a representation of bulk unit weight or particle solid unit weight?
For 1st Condition. Considering a unit weight of 19.62 kN/m3 for dry soil (see table of parameters) and a maximum deph of 20 m, a maximum value of total stress of 392.4 kPa  is obtained in comparing with results of Anura3D model (374 kPa). What could it be the reason for different results?
For 2nt condition. Is it correct that only an unique value of 19.62 kPa is obtained despite two materials (dry and satured soils) have been modelled?
If It is introduced a horizontal phreatic level in the model, why wasn´t it develope water pressure under phreatic level?

Thank you in advance for your help.

**Comment by luisez1988** [comment, marked as answer]
Hi, 

This is an issue of the output of pore pressure not being generated because you assume drained behavior. We do not write the pore pressure because, for the submerged material, effective stresses and total stresses are calculated based on using buoyant unit weights. As a result, it results in the output showing zero pore pressure and the two stress fields being equal. The calculation is correct regarding effective stresses, so total stresses and pore pressure are meaningless. 

Also, you do not need the water table definition because that is required only when you have unsaturated conditions and want to model a continuous pore pressure field in a soil layer.

This needs to be corrected in our code so the output is consistent. It is on our list now.
